generate-stackbrew-library.sh

```diff
@@ -3,8 +3,8 @@ set -Eeuo pipefail
 
 # https://www.haproxy.org/#down ("LTS" vs "latest")
 declare -A aliases=(
-	[2.5]='latest'
-	[2.4]='lts'
+	[2.6]='latest'
+	[2.6]='lts'
```

Comment:
For this to work correctly (and not end up with just `lts`) we need to join these back together:

```suggestion
	[2.6]='lts latest'
```

(Basically the reverse of the change in https://github.com/docker-library/haproxy/commit/50b6692229b0d82c35f925548c8cd24111319d18#diff-9dd720074107975da3d3140d5e27deb1e3870cb10fd74029ded8d6e1efe98346 :smile:)

Reply:
Thanks for the review. I'm already off the computer for tonight. Feel free to commit into the PR branch. Otherwise I'll handle the changes tomorrow.